### DISCUSSION TITLE
I want propose add-on bar to have the redirect tooltip inside

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [Aris-t2/CustomJSforFx#119] [Ideas]
I want to propose an improvement of the add-on bar by making the redirect tooltip/link hover tooltip inside the add-on bar but as a movable element.

Here is how should it be solved:
![Screenshot 2024-08-07 165014](https://github.com/user-attachments/assets/ca306d15-cb59-45f6-991a-0c543ad5ac28)

**Comment by Aleksandra0056** [comment]
For example, the Status Bar script from firefox-scripts by xiaoxiaoflood (https://github.com/xiaoxiaoflood/firefox-scripts) already has this behaviour.

**Reply by rebmcr** [reply]
I've submitted a PR which fixes this

https://github.com/Aris-t2/CustomJSforFx/pull/135

**Reply by Aris-t2** [reply]
The code was there all the time. It is not active on purpose, but everybody can enable it, if needed. No need to make this the default behavior. Not need to "fix" this.

**Reply by rebmcr** [reply]
Without the extra block modifying the gBrowser function, it 'undocks' every time the active tab changes.
